scripts/keri/cf/verifier-config-public.json

```
@@ -29,5 +29,9 @@
     "https://gleif-it.github.io/oobi/EH6ekLjSr8V32WyFbGe1zXjTzFs9PkTYmupJ9H65O14g",
     "https://gleif-it.github.io/oobi/EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao"
   ],
-  "LEIs": []
+  "LEIs": [],
+
+  "acceptedRoles": [
```

We will have accepted roles for ECRs, OORs, and maybe other creds. I have recommended ecrRoles
```suggestion
  "ecrRoles": [
```